core/frontend/src/tile/map/BingElevation.ts

```diff
@@ -46,6 +48,10 @@ export class BingElevationProvider {
     this._seaLevelOffsetRequestTemplate = "https://dev.virtualearth.net/REST/v1/Elevation/SeaLevel?points={points}&key={BingMapsAPIKey}".replace("{BingMapsAPIKey}", bingKey);
     this._heightListRequestTemplate = "https://dev.virtualearth.net/REST/v1/Elevation/List?points={points}&heights={heights}&key={BingMapsAPIKey}".replace("{BingMapsAPIKey}", bingKey);
   }
+
+  /** Return the height (altitude) at a given cartographic location
```

Comment:
Can you explain what `geodetic` does here and elsewhere?

Reply:
Yes, done.